packages/core/test/integration/query-interface/add-show-remove-index.test.ts

```diff
@@ -0,0 +1,304 @@
+'use strict';
```

Comment:
no need to use strict in ts files, the transpiler adds it

Reply:
Updated now.